packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MyAppPackage.kt

```diff
@@ -0,0 +1,19 @@
+package renative.harness.debug
```

Comment:
why is this in the android platform template? It's called MyPackage and references harness

Reply:
templatePath is now hardcoded. I can bring back a structure based on appId. Are you sure we need such a complex folder structure?